frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -136,7 +137,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       detect: detectComponents,
     },
     flags: {
-      required: [OCS_FLAG],
+      required: [ODF_MODEL_FLAG],
```

Comment:
Can we use `OCS_MODEL_FLAG` in case someone is using OCS 4.8 ??
Or it is not allowed to use OCS 4.8 on OCP 4.10 ??

Reply:
OCP 4.10 will not support OCS 4.8